src/collection/components/Cart/Cart.tsx

```diff
@@ -27,12 +28,43 @@ const Cart: React.FC<Props> = (props: Props) => {
       });
       return value;
     }
+
+    const handleItemClick = (item: CollectionItemModel) => {
+      if(item.tier !== CollectionItemTier.TIER1) return;
+      
+      const latestItem = cartList.find(x => x.collectionPhase === CollectionPhase.LATEST);
+      
+      if(latestItem?.tier !== CollectionItemTier.TIER1) return;
+
+      displayWarning();
+    }
+
+    const displayWarning = async () => {
```

Comment:
Is there any reason that this method is `async`?

Reply:
IMO if this method is executed synchronously, it will cause performance degradation to the renderer process.

Reply:
I want to run this on another task, not renderer.

Reply:
As far I know, [`setTimeout()`](https://developer.mozilla.org/ko/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) doesn't block. it just adds a timer to the main event thread.